README.md

```diff
@@ -1,66 +1,66 @@
-# README
+![Logo](https://cdn.awork.com/img/logo/blue/complete/blue-complete.svg)
 
-![Logo](https://www.awork.io/wp-content/uploads/2019/01/aworkLogoB-1.svg)
+[awork](https://www.awork.com/) is an awesome work management tool that helps you to be more productive. Next to a [web app](https://app.awork.com/) and an app for [iOS](https://apps.apple.com/de/app/awork-organisiere-dein-team/id1466945183) and [Android](https://play.google.com/store/apps/details?id=io.awork\&hl=gsw\&gl=US), we also provide a rich API that enables external developers to integrate awork into their workflows. Here you can find more details, FAQ's, code examples and a help forum to get you started even faster.
```

Comment:
https://play.google.com/store/apps/details?id=io.awork\&hl=gsw\&gl=US

On Google Play Store we are still going to be io.awork?

Reply:
Yeah these are unique identified that we're not changing.